src/DataMiner/SLNetTypes/SubscriptionFilters/SubscriptionFilter.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+
+using Skyline.DataMiner.Net.Messages;
+
+namespace Skyline.DataMiner.Net
+{
+	/// <summary>
+	/// <para>SubscriptionFilters are used in CreateSubscriptionMessage to indicate which types of messages
+	/// should be subscribed to. The different types of <see cref="SubscriptionFilter"/> objects can be used 
+	/// to subscribe with a certain level of granularity:</para>
+	/// <list type="bullet">
+	///		<item><description>
+	///			<see cref="SubscriptionFilter"/>: subscribes to all messages of a given type (DMS)
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterCluster: limits subscription to one particular cluster 
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterDMA: limits messages to one particular DataMiner Agent
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterElement: limits messages to one particular element
+	///		</description></item>
+	///		<item><description>
+	///			SubscriptionFilterParameter: limits messages to one particular parameter of a given parameter
```

Comment:
Is this correct? "a parameter of a parameter" seems a bit odd.

Reply:
updated